src/style.css

```diff
@@ -2,2 +1,85 @@
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to add fallback fonts. If the Roboto font fails to load for some reason, the browser will use the generic sans-serif font. It's better to specify one or two fallback fonts that are similar to Roboto.